swarm/network/stream/delivery_test.go

```diff
@@ -306,7 +306,7 @@ func TestStreamerDownstreamChunkDeliveryMsgExchange(t *testing.T) {
 
 }
 
-func TestDeliveryFromNodes(t *testing.T) {
+func XTestDeliveryFromNodes(t *testing.T) {
```

Comment:
I haven't notice that this test is failing recently, but certainly should be checked thoroughly.